newsroom/wire/views.py

```diff
@@ -304,6 +304,12 @@ def download(_ids):
         _file.seek(0)
         mimetype = formatter.get_mimetype(item)
         attachment_filename = secure_filename(formatter.format_filename(item))
+    elif formatter.FILE_EXTENSION == "csv" and len(items) != 1:
```

Comment:
would be better if the formatter can define it can handle multiple files instead of hardcoding it here for specific one, eg.:
```py
if formatter.multi is True:
    data, filename = formatter.format_items(...)
```

Reply:
got it, Thanks !